src/main/java/cinema/model/User.java

```diff
@@ -15,6 +19,8 @@
     @Column(unique = true)
     private String email;
     private String password;
+    @OneToMany(cascade = CascadeType.PERSIST, fetch = FetchType.EAGER)
+    private Set<Role> roles;
```

Comment:
Can each role belone to one user only? 
fetchType eager is bad practice

Reply:
done